Cargo.toml

```diff
@@ -18,6 +18,11 @@ bitflags = "1.0"
 cfg-if = "0.1.0"
 void = "1.0.2"
 
+# Don't include the optional serde feature
```

Comment:
Can you move this comment to right about the `default-features` line so it's more clear that's what it's pertaining to?